modules/exploits/multi/http/manageengine_servicedesk_plus_saml_rce_cve_2022_47966.rb

```diff
@@ -213,7 +281,10 @@ def execute_command(cmd, _opts = {})
       }
     })
 
-    unless res&.code == 500
+    # Java payload returns a nil response on successful execution of payload
+    if target['Type'] == :java && res.nil?
+      print_status('Exploit completed. New session must be opened by now else exploit failed.')
```

Comment:
Not every Java exploit opens a session.

```suggestion
      print_status('Exploit completed.')
```

Reply:
Changed message accordingly